web/modules/custom/feedback/feedback.routing.yml

```diff
@@ -1,11 +1,3 @@
-feedback.content:
-  path: '/feedback'
-  defaults:
-    _controller: 'Drupal\feedback\Controller\FeedbackController::content'
-    _title: 'Feedback'
-  requirements:
-    _permission: 'access content'
-
```

Comment:
this route and controller was created to accommodate for feedback block. it was convenient for developments. now I delete it.